src/openrct2/localisation/LanguagePack.cpp

```diff
@@ -198,15 +198,14 @@ class LanguagePack final : public ILanguagePack
         }
     }
 
-    rct_string_id GetObjectOverrideStringId(const char* objectIdentifier, uint8_t index) override
+    rct_string_id GetObjectOverrideStringId(const std::string_view& objectIdentifier, uint8_t index) override
```

Comment:
This takes a legacy identifier, right? If so: perhaps best to change the parameter name.

Reply:
sure